src/main/plugins/org.dita.pdf2/xsl/fo/loc.xsl

```diff
@@ -0,0 +1,256 @@
+
+    <!-- Create a list of all book releases -->
+    <xsl:variable name="bookReleaseDates" select="*[contains (@class, ' topic/created ')]/@date,
+                                                  *[contains (@class, ' topic/revised ')]/@modified"/>
+    <xsl:variable name="bookReleaseString">
+      <xsl:call-template name="getVariable">
+        <xsl:with-param name="id" select="'Book Release'"/>
+      </xsl:call-template>
+    </xsl:variable>
+    <xsl:variable name="bookReleaseDateString">
+      <xsl:call-template name="getVariable">
+        <xsl:with-param name="id" select="'Book Release Date'"/>
+      </xsl:call-template>
+    </xsl:variable>
+    <xsl:variable name="colonSymbol">
+      <xsl:call-template name="getVariable">
+        <xsl:with-param name="id" select="'ColonSymbol'"/>
+      </xsl:call-template>
+    </xsl:variable>
+    <xsl:variable name="separator">
+      <xsl:call-template name="getVariable">
+        <xsl:with-param name="id" select="'figure-number-separator'"/>
+      </xsl:call-template>
+    </xsl:variable>
+
+    <xsl:for-each select="1 to (count($bookReleaseDates) - 1)">
+      <xsl:variable name="i" select="position()"/>
+
+      <!-- timeRangeStart and timeRangeEnd set the date range between two book releases -->
+      <xsl:variable name="timeRangeStart" select="$bookReleaseDates[$i]"/>
+      <xsl:variable name="timeRangeEnd" select="$bookReleaseDates[$i + 1]"/>
+      <xsl:variable name="timeRangeStartAsDate" select="xs:date($timeRangeStart)"/>
+      <xsl:variable name="timeRangeEndAsDate" select="xs:date($timeRangeEnd)"/>
+
+      <fo:block>
+        <fo:table xsl:use-attribute-sets="releaseManagementTable">
+          <xsl:call-template name="selectAtts"/>
+          <xsl:call-template name="globalAtts"/>
+          <fo:table-body>
+            <fo:table-row xsl:use-attribute-sets="releaseManagementTable.bookRelease.row">
+              <fo:table-cell xsl:use-attribute-sets="releaseManagementTable.bookRelease.cell">
+                <fo:block xsl:use-attribute-sets="releaseManagementTable.bookRelease.content">
+                  <fo:block>
+                    <xsl:value-of select="$bookReleaseString"/>
+                    <xsl:value-of select="$colonSymbol"/>
+                    <xsl:value-of select="$separator"/>
+                    <xsl:value-of select="$i"/>
+                  </fo:block>
+                  <fo:block>
+                    <xsl:value-of select="$bookReleaseDateString"/>
+                    <xsl:value-of select="$colonSymbol"/>
+                    <xsl:value-of select="$separator"/>
+                    <xsl:value-of select="format-date($timeRangeEndAsDate,'[MNn] [D], [Y]')"/>
+                  </fo:block>
+                </fo:block>
+              </fo:table-cell>
+            </fo:table-row>
+            <!-- Iterate over all change-items -->
+            <xsl:for-each select="1 to (count($changeset))">

```

Comment:
This is a counter based for-loop and not suitable style in out XSLT. Use a normal for-each to process each `$changeset` node, thus `$currentChangeItem` becomes the current node.